dsl/jpql/src/main/kotlin/com/linecorp/kotlinjdsl/dsl/jpql/Jpql.kt

```diff
@@ -536,6 +536,56 @@ open class Jpql : JpqlDsl {
         return Expressions.div(this.toExpression(), value.toExpression())
     }
 
+    /**
+     * Creates an expression that represents the mod of values.
+     * The values are each enclosed in parentheses.
+     *
+     * This is the same as ```(value1) % (value2)```.
+     */
+    @JvmName("modWithParentheses")
+    @SinceJdsl("3.3.2")
+    fun <T : Number, S : T?> mod(value1: Expressionable<@Exact T>, value2: S): Expression<T> {
+        return Expressions.mod(
+            Expressions.parentheses(value1.toExpression()),
+            Expressions.parentheses(Expressions.value(value2)),
+        )
+    }
+
+    /**
+     * Creates an expression that represents the mod of values.
+     * The values are each enclosed in parentheses.
+     *
+     * This is the same as ```(value1) % (value2)```.
+     */
+    @JvmName("modWithParentheses")
+    @SinceJdsl("3.3.2")
+    fun <T : Number, S : T> mod(value1: Expressionable<@Exact T>, value2: Expressionable<S>): Expression<T> {
+        return Expressions.mod(
+            Expressions.parentheses(value1.toExpression()),
+            Expressions.parentheses(value2.toExpression()),
+        )
+    }
+
+    /**
+     * Creates an expression that represents the mod of values.
+     *
+     * This is the same as ```value1 % value2```.
+     */
+    @SinceJdsl("3.3.2")
+    fun <T : Number, S : T?> Expressionable<@Exact T>.mod(value: S): Expression<T> {
```

Comment:
There was something I missed. [In JPQL](https://jakarta.ee/specifications/persistence/3.1/jakarta-persistence-spec-3.1.html#a5284), mod is not an operator, so I think it would be better to provide it as a function to reduce confusion.

So, unfortunately, I'd like to ask you to remove the extension function.